Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -457,10 +460,24 @@ def retrieve_and_display_info_and_open_pr(formula_or_cask, name, repositories, a
                                       #{outdated_synced_formulae.join(", ")}.
           EOS
         end
-        puts <<~EOS unless args.no_pull_requests?
-          Duplicate pull requests:       #{duplicate_pull_requests       || "none"}
-          Maybe duplicate pull requests: #{maybe_duplicate_pull_requests || "none"}
-        EOS
+        unless args.no_pull_requests?
+          if versions_equal
+            duplicate_pull_requests_text = "skipped"
```

Comment:
This explicitly outputs "Duplicate pull requests:  skipped" when the versions are equal, to make it clear that `bump` is working as intended and the check was intentionally skipped (i.e., not a bug with `--no-pull-requests` handling).

If it's fine to omit the duplicate PR output when versions are equal, I can update this accordingly. We have over ~3,200 formulae in the homebrew/core autobump list, so there's something to be said for omitting the PR output for the ~98% of packages that are up to date in a typical run.